### DISCUSSION TITLE
Self-hosted runners aren't used when max allowed time is reached

### DISCUSSION THREAD

**Discussion by bubthegreat** [community/community#25584] [Actions]
<p>We ran out of free time on our actions runs, so I found the actions beta for self-hosted runners was live - but right now I’m still getting the same error about billing status even though I’ve updated my workflows to run on self-hosted and the runner has been successfully added to the repository.</p>
<p>This looks like a bug where it checks the time limit first before if it checks if there are available self-hosted runners - but I’d like confirmation before I start getting crazy.</p>
<p>For reference, here’s the runner status:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9c11f091c4a665ae755bedde56b85b374a2f795b.png" data-download-href="/uploads/short-url/mgEV6Bf8cTHG1Dp4FN85jlE5wEP.png?dl=1" title="Annotation 2019-11-15 121916.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12198817/181091892-91b44824-f5ec-43d8-b6b8-fd5804ccb6fc.png" alt="Annotation 2019-11-15 121916.png" data-base62-sha1="mgEV6Bf8cTHG1Dp4FN85jlE5wEP" width="690" height="187" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/9c11f091c4a665ae755bedde56b85b374a2f795b_2_690x187.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/9c11f091c4a665ae755bedde56b85b374a2f795b_2_1035x280.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/9/9c11f091c4a665ae755bedde56b85b374a2f795b_2_1380x374.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9c11f091c4a665ae755bedde56b85b374a2f795b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2019-11-15 121916.png</span><span class="informations">1682×458 18.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The workflow file portion that’s pertinent:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/11a39bff197b001eaa3bcf8916aadd23d4e3ee96.png" data-download-href="/uploads/short-url/2w2EcAOmZLEcGKkdmRwldnLY4U6.png?dl=1" title="Annotation 2019-11-15 121955.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12198817/181091893-2ac174f6-9e71-4caa-8b44-53ea554f6d61.png" alt="Annotation 2019-11-15 121955.png" data-base62-sha1="2w2EcAOmZLEcGKkdmRwldnLY4U6" width="637" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/11a39bff197b001eaa3bcf8916aadd23d4e3ee96_2_637x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/11a39bff197b001eaa3bcf8916aadd23d4e3ee96_2_955x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/11a39bff197b001eaa3bcf8916aadd23d4e3ee96.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/11a39bff197b001eaa3bcf8916aadd23d4e3ee96_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2019-11-15 121955.png</span><span class="informations">1036×812 31.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>and the error:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9844fbec0da3933a9fd1d85e03a8f91226ce9fa3.png" data-download-href="/uploads/short-url/lJ2n8zwstyFd1xXwlNdBUcZq5az.png?dl=1" title="Annotation 2019-11-15 121955-2.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12198817/181091894-c528546d-3dbf-43c5-b0fb-438ed19d3b90.png" alt="Annotation 2019-11-15 121955-2.png" data-base62-sha1="lJ2n8zwstyFd1xXwlNdBUcZq5az" width="690" height="145" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/9844fbec0da3933a9fd1d85e03a8f91226ce9fa3_2_690x145.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/9844fbec0da3933a9fd1d85e03a8f91226ce9fa3_2_1035x217.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/9/9844fbec0da3933a9fd1d85e03a8f91226ce9fa3_2_1380x290.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9844fbec0da3933a9fd1d85e03a8f91226ce9fa3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2019-11-15 121955-2.png</span><span class="informations">1780×376 15.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>And </p>

**Comment by mscoutermarsh** [comment]
<p>Hi <a class="mention" href="/bubthegreat">@bubthegreat</a>, which account is this happening under? </p>

**Comment by bubthegreat** [comment]
<p><a href="https://github.com/artemishealth/query_parser" rel="nofollow noopener">https://github.com/artemishealth/query_parser</a></p>
<p>It’s private for artemishealth, I’m a member of the org, and I’m an admin of the repo.  </p>
<p>Specifically the branch that we’re setting up these builds on:</p>
<p><a href="https://github.com/artemishealth/query_parser/tree/feature/python-parser-build" rel="nofollow noopener">https://github.com/artemishealth/query_parser/tree/feature/python-parser-build</a></p>
<p>That has the three workflows that are defined with self-hosted runner - the runner is assigned to that project and still shows as idle/reachable.</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Hi <a class="mention" href="/bubthegreat">@bubthegreat</a>,</p>
<p>I looked into it.</p>
<p>It looks like the organization would need to make some changes to the current billing plan in order to re-enable Actions on the account. Please have an organization owner or billing manager reach out for more details and we’ll be glad to help. </p>
<p><a href="https://support.github.com/contact" rel="nofollow noopener">https://support.github.com/contact</a></p>
